yarn.lock

```diff
@@ -2454,6 +2454,13 @@
   dependencies:
     babel-helper-vue-jsx-merge-props "2.0.3"
 
+"@justeat/f-vue-icons@3.4.0":
```

Comment:
Not quite sure why it isn't at 3.5.0... looking into this

Reply:
@ashleynolan @dandel10n I've tried a number of times to reinstall yarn but it never appears to update the lock file to `3.5.0`. Is this a concern?

Reply:
Thank you!